Sources/TinyNetworking/URLEncoding.swift

```diff
@@ -37,13 +37,20 @@ public struct URLEncoding {
             }
         }
     }
+    
+    public enum Destination {
+        case urlQuery
+        case httpBody
+    }
 
     public let arrayEncoding: ArrayEncoding
     public let boolEncoding: BoolEncoding
+    public let destination: Destination
 
-    public init(arrayEncoding: ArrayEncoding = .brackets, boolEncoding: BoolEncoding = .literal) {
+    public init(destination: Destination = .urlQuery, arrayEncoding: ArrayEncoding = .brackets, boolEncoding: BoolEncoding = .literal) {
```

Comment:
Does it make sense to make `destination` optional? 
Optional would mean that if it's `nil`, the destination would be inferred from the `HTTP` method. WDYT?

Reply:
Sounds good, let me change it and push a new commit :)

Reply:
@jdisho pushed the changes